src/main/java/net/imagej/ops/sieve/highSieve/DefaultHighSieve.java

```diff
@@ -0,0 +1,25 @@
+package net.imagej.ops.sieve.highSieve;
+
+import net.imagej.ops.Ops;
+import net.imagej.ops.special.function.AbstractBinaryFunctionOp;
+import net.imglib2.type.Type;
+import org.scijava.plugin.Plugin;
+
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+/**
+ * Creates an Iterable from all the elements in the input that are greater than the given value
+ *
+ * @author Richard Domander (Royal Veterinary College, London)
+ */
+@Plugin(type = Ops.Sieve.HighSieve.class)
+public class DefaultHighSieve<T extends Type<T>, S extends Comparable<T>>
+        extends AbstractBinaryFunctionOp<Iterable<T>, S, Iterable<T>>
```

Comment:
Ops uses tabs instead of spaces according to the [Coding Style](http://imagej.net/Coding_style).